src/format.ts

```diff
@@ -10,6 +10,7 @@ const enum Pillar {
 
 const enum Special {
 	SpecialReport = 5,
+	Labs = 6,
```

Comment:
Adding this value in here _seemed_ like the right place (rather than adding a new const enum) but it's not ideal.

I'm not sure if there are any additional pillars that might appear or if this is an area we might need to extend in future meaning we want to think about this vocabulary more?